### DISCUSSION TITLE
Add HTML sanitizer tags for WYSIWYG

### DISCUSSION THREAD

**Discussion by giorgiof** [pimcore/pimcore#16036] [Q&A / Support] [answered]
should h2-h6 tag also be added to
pimcore.wysiwyg_sanitizer:
allow_elements
?
(ref: https://github.com/pimcore/pimcore/commit/91811b8b8214917e71027f90de90a4426abc9395)

**Comment by robertSt7** [comment, marked as answer]
@giorgiof Thanks for reporting this. I have created a PR for that https://github.com/pimcore/pimcore/pull/16099
